src/ECER.Clients.RegistryPortal/ecer.clients.registryportal.client/src/components/inputs/EceRadio.vue

```diff
@@ -1,15 +1,35 @@
 <template>
-  <v-radio-group>
-    <v-radio label="Radio One" value="one"></v-radio>
-    <v-radio label="Radio Two" value="two"></v-radio>
-    <v-radio label="Radio Three" value="three"></v-radio>
+  <div>
+    <h3>
+    {{ props.title }}
```

Comment:
Do you think, the title for the radio group should be inside or outside component?

Reply:
I think it should go inside the component. Since we can set it via props.

Reply:
This is how we do that now, it is in input and we set its value from props. do you mean this is good?

Reply:
yep this is good.